modules/runtime/pages/process-manager-application.adoc

```diff
@@ -0,0 +1,60 @@
+= Creating a process manager application
+:description: This page explains how to create your own custom process manager application
+
+[NOTE]
+====
+For Subscription editions only. +
+If your Bonita Subscription has been purchased before 2021, this Add-on can be purchased on top of your subscription.
+If you have purchased or renewed your Bonita Subscription in 2021 or after, BCD is fully part of your subscription.
+====
+
+This page explains how to create your own custom process manager application. +
+This application functions like the Administrator application, but has a smaller scope, one that only controls processes, cases and tasks. +
+Creating such an application follows the same steps as the creation of any xref:applications:application-creation.adoc[other custom application].
+
+== Pages
+
+The main menu of the application will require four pages.
+
+* Monitoring
+* Process list
+* Case list
+* Task list
+
+In addition to those, there are some orphan pages that are required by the above ones :
+
+* Process details
+* Process visualization
+* Case details
+* Case visualization
+* Task details
+
+== Page modifications
+
+The above-mentioned menu pages are the same as the ones that are part of the Administrator application, but they only show the processes/cases/tasks that the user supervises. +
+In order to make the menu pages do so, you need to export them from the Administrator Resource List and edit them using the UI Designer.
+You don't need to export and modify the orphan pages.
+
+[WARNING]
+====
+
+Currently, the Monitoring page is made using a different technology than UI Designer, and thus it cannot be edited.
+====
+
+The modifications that you will need to do are quite simple:
+
+1. Though not necessary, I highly recommend re-saving your pages with a different name. This will prevent the pages from being over-written if you need to re-import the default pages.
+2. Add an api call to get current user session. Create a new "External API" variable called `session` with the value `../API/system/session/unusedId`
+3. Add `&f=supervisor_id={{session.user_id}}` to the variables of the "External API" type (except for the `session` variable that we created).
+
+== Application creation
+
+After all the menu pages have been modified, we can proceed with the application creation. We can create the application from scratch, but it's easier to : +
+
+1. Export the Bonita Administrator Application from the Application List and import it into the Bonita Studio
+2. Delete any orphan page that isn't mentioned in the list above
+3. Delete all, but the BPM menu from the "Navigation" section
+4. Create application pages in the "Navigation" menu using the menu pages that we mentioned above. Make sure to keep the tokens of the pages as the ones from the Administrator application, since the pages make use of the specific tokens or, if you modify the tokens, modify the pages to also use the correct urls.
```

Comment:
This sentence too long, it's hard to read. 
And you start a sentence with 'make sure to..' for something that is recommended, but according to the end of the sentence not mandatory, it feels weird, I suggest to say it differently.

```suggestion
4. Create application pages in the "Navigation" menu using the menu pages mentioned above. It is highly recommended to keep the tokens of the pages as the ones from the Administrator application, since the pages make use of the specific tokens. If you modify these tokens, make sure to modify the pages to also use the correct urls.
```